third_party/2and3/click/utils.pyi

```diff
@@ -90,22 +67,17 @@ def get_text_stream(
 ) -> IO[str]:
     ...
 
-
 def open_file(
     filename: str,
     mode: str = ...,
     encoding: Optional[str] = ...,
     errors: str = ...,
     lazy: bool = ...,
     atomic: bool = ...
-) -> Union[IO, LazyFile, KeepOpenFile]:
-    ...
-
-
+) -> Union[IO[AnyStr], LazyFile[AnyStr], KeepOpenFile[AnyStr]]: ...
```

Comment:
This is an unconstrained type variable (as well as a union return type). With a lot of overloads you can probably do it right, but I'd be fine with falling back to Any.

Reply:
Whoops, `Union` != `Tuple`. I'll just use `Any` for now as suggested.